packages/frontend-2/plugins/accessibility.ts

```diff
@@ -0,0 +1,5 @@
+import { vKeyboardClickable } from '@speckle/ui-components'
+
+export default defineNuxtPlugin((nuxtApp) => {
+  nuxtApp.vueApp.directive('keyboard-clickable', vKeyboardClickable)
```

Comment:
Ideally the ui-components-nuxt package should do this, but we can do it separately

Reply:
I'll create tickets to follow-up on this and the other issues :)